sw/host/opentitanlib/src/bootstrap/mod.rs

```diff
@@ -116,6 +117,17 @@ impl<'a> Bootstrap<'a> {
         transport: &TransportWrapper,
         options: &BootstrapOptions,
         payload: &[u8],
+    ) -> Result<()> {
+        Self::update_with_progress(transport, options, payload, |_, _| {})
+    }
+
+    /// Perform the update, sending the firmware `payload` to a SPI or UART target depending on
+    /// given `options`, which specifies protocol and port to use.
+    pub fn update_with_progress(
+        transport: &TransportWrapper,
+        options: &BootstrapOptions,
+        payload: &[u8],
+        progress: impl Fn(u32, u32),
```

Comment:
Random: I just noticed this, but shouldn't our Rust style guide include doc comments for parameters of functions just like our C style guide?